schema/contributor-object.schema.json

```diff
@@ -1,34 +1,41 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://w3c.github.io/wpub/schema/contributor-object.schema.json",
-  "title": "Contributor Object",
-  "type": "object",
-  "properties": {
-    "name": {
-      "$ref": "localizable.schema.json"
-    },
-    "id": {
-      "type": "string",
-      "format": "uri"
-    },
-    "type": {
-      "anyOf": [
-        {
-          "type": "string",
-          "enum": ["Person", "Organization"]    
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://w3c.github.io/pub-manifest/schema/contributor-object.schema.json",
+    "title": "Contributor Object",
+    "type": "object",
+    "properties": {
+        "name": {
+            "$ref": "localizable.schema.json"
         },
-        {
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
-          "contains": {
-            "enum": ["Person", "Organization"]
-          } 
+        "id": {
+            "type": "string",
+            "format": "uri"
+        },
+        "type": {
+            "anyOf": [
+                {
+                    "type": "string",
+                    "enum": [
+                        "Person",
+                        "Organization"
+                    ]
```

Comment:
I am not sure what that would mean... The constraint says that the value most be a string, either 'Person' or 'Organization', and does not allow for a non-empty value. What would a "default" add to that constraint?

Reply:
I'm thinking of the compact case where all you have is a string. Yes, the type can't be empty, but of the two we say that Person is the default when absent. I noticed that we specify the default readingProgression as ltr, so if we're including that default why not others?

 (But defaults are somewhat useless, so this is more an exercise in being a bit more self-documenting within the schema.)

Reply:
Yeah, the 'ltr' was there and I wasn't sure either why... But o.k. for consistency and self-documentation... so I have just added it.

Reply:
I don't see why there should be a default contributor type. By default we don't know, this is a proper RDF conclusion, even if bad practice in JSON-LD land. 
This is different from readingProgression, for which without default we (the RS) cannot act, which is more embarrassing.

More generically, I think that we (the WG) shouldn't care about the contributor's type(s) at the JSON schema level and should only constraint its value space via "type": [ "string",  "array"]

Reply:
> More generically, I think that we (the WG) shouldn't care about the contributor's type(s) at the JSON schema level and should only constraint its value space via "type": [ "string", "array"]

I think this is not a JSON Schema issue. If we use it, it should reflect the possible 'shape' of the manifest. The spec does say that, e.g., the contributor is an organization _or_  person right now, so that is what the Schema says. If we decide to drop it from the manifest, we should drop it from the Schema, too.